src/modules/status.rs

```diff
@@ -279,6 +285,28 @@ mod tests {
         }
     }
 
+    #[test]
+    fn failure_hex_status() {
+        let exit_values = [1, 2, 130, -2147467260];
+
+        for status in exit_values.iter() {
```

Comment:
Can we create a second array that matches `exit_values` (maybe called `string_values`) that contains the actual string values of the outputs we expect to see?

As someone who doesn't remember what the `:X` formatter does off the top of my head (and likely will not in the foreseeable future), I'm currently having to scramble through the docs for std::format trying to find out what that format string does. This seems like a great opportunity to add some examples in the code itself.

Reply:
I changed it accordingly, thanks for the review!